staff/groups/groups/ff-productions/film-fest/film-fest-doc/template.html

```diff
@@ -160,19 +160,24 @@ <h2 class="specs__title">Movie</h2>
         //pending personal area, watch list, mody profile, favourite section
 
     </main>
-    <footer>
-        <nav class="footer-nav">
 
-            <p class="footer-nav__go-up">Go up!</p>
+    <footer class= "bottom">
+        <nav class="footer-nav">
+                <div class="footer-nav go-up">

```

Comment:
there is an bad praxis on ben construction; 
div is a footer-nav child so it need to be like : "footer-nav__go-up go-up" instead of  "footer-nav go-up"

Reply:
Ok.. Done!! Thank you